### PR TITLE
[MI-2688]:Fixed review comments by mm team for issue #500 on github plugin

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
There are some additional review comments
https://github.com/mattermost/mattermost-plugin-github/pull/632

### CODE REVIEW

File: server/plugin/plugin.go [hunk 1]
```diff
@@ -113,6 +113,12 @@ func NewPlugin() *Plugin {
 		"issue":         p.handleIssue,
 	}
 
+	p.createGithubEmojiMap()
+
```
Comment (raghavaggarwal2308) [inline server/plugin/plugin.go]:
```suggestion
```

**Review by raghavaggarwal2308 — APPROVED** [review]

File: server/plugin/plugin.go [hunk 2]
```diff
@@ -281,6 +285,7 @@ func (p *Plugin) getPostPropsForReaction(reaction *model.Reaction) (orgRepo []st
 		return orgRepo, id, objectType, false
 	}
 
+	// Getting the github repository from notification post props
```
Comment (ayusht2810) [inline server/plugin/plugin.go]:
```suggestion
	// Getting the GitHub repository from notification post props
```

Comment (ayusht2810) [reply server/plugin/plugin.go]: maybe? if changing, change everywhere

File: server/plugin/plugin.go [hunk 3]
```diff
@@ -308,6 +315,7 @@ func (p *Plugin) getPostPropsForReaction(reaction *model.Reaction) (orgRepo []st
 func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
 	githubEmoji := p.emojiMap[reaction.EmojiName]
 	if githubEmoji == "" {
+		p.API.LogWarn("Emoji is not supported by github", "emoji", reaction.EmojiName)
```
Comment (ayusht2810) [inline server/plugin/plugin.go]:
```suggestion
		p.API.LogWarn("Emoji is not supported by github", "Emoji", reaction.EmojiName)
```

File: server/plugin/plugin.go [hunk 4]
```diff
@@ -350,6 +358,7 @@ func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reactio
 func (p *Plugin) ReactionHasBeenRemoved(c *plugin.Context, reaction *model.Reaction) {
 	githubEmoji := p.emojiMap[reaction.EmojiName]
 	if githubEmoji == "" {
+		p.API.LogWarn("Emoji is not supported by github", "emoji", reaction.EmojiName)
```
Comment (ayusht2810) [inline server/plugin/plugin.go]: same

**Review by ayusht2810 — APPROVED** [review]

File: server/plugin/plugin.go [hunk 5]
```diff
@@ -358,25 +360,26 @@ func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reactio
 func (p *Plugin) ReactionHasBeenRemoved(c *plugin.Context, reaction *model.Reaction) {
 	githubEmoji := p.emojiMap[reaction.EmojiName]
 	if githubEmoji == "" {
-		p.API.LogWarn("Emoji is not supported by github", "emoji", reaction.EmojiName)
+		p.API.LogWarn("Emoji is not supported by github", "Emoji", reaction.EmojiName)
```
Comment (ayusht2810) [inline server/plugin/plugin.go]:
```suggestion
		p.API.LogWarn("Emoji is not supported by GitHub", "Emoji", reaction.EmojiName)
```

File: server/plugin/plugin.go [hunk 6]
```diff
@@ -278,62 +277,65 @@ func (p *Plugin) OnDeactivate() error {
...
 
-	return orgRepo, id, objectType, true
+	return org, repo, id, objectType, true
 }
 
 func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
 	githubEmoji := p.emojiMap[reaction.EmojiName]
 	if githubEmoji == "" {
-		p.API.LogWarn("Emoji is not supported by github", "emoji", reaction.EmojiName)
+		p.API.LogWarn("Emoji is not supported by github", "Emoji", reaction.EmojiName)
```
Comment (ayusht2810) [inline server/plugin/plugin.go]:
```suggestion
		p.API.LogWarn("Emoji is not supported by GitHub", "Emoji", reaction.EmojiName)
```

**Review by ayusht2810 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**